src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,15 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        StringBuilder stringBuilder = new StringBuilder();
+        if (value > 1) {
+            while (value > 0) {
+                int s = !(value % 2 == 0) ? 1 : 0;
```

Comment:
this variable is redundant, try to pass value directly and without using ternary operator

Reply:
done